application/migrations/007_add_omb_monitored_flag.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Migration_add_omb_monitored_flag extends CI_Migration
+{
+
+    public function up()
+    {
+        // Add a column to indicate whether the office is one that the OMB is explicitly monitoring
```

Comment:
It's kind of funny that the migration is really the only place that makes sense to document the data model.

I might be a little more specific here. What does it mean for OMB to be monitoring an office?

> Add a column to indicate whether the office is one that OMB is explicitly monitoring for an agency's progress on compliance of laws and regulations e.g. OPEN Government Data Act.

Reply:
That's just it... We don't know yet! Hyon said she thinks it may indicate that the office already has a CDO in place, but until I know for sure that's what this flag refers to, I wanted to be non-specific.